ERROR - 2024-05-18 12:49:59 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for db5013982481.hosting-data.io failed: No such host is known.  in E:\en desarrollo\upn212evaldoc\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'E:\\en desarroll...', 173)
#1 E:\en desarrollo\upn212evaldoc\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('db5013982481.ho...', 'dbu1124368', Object(SensitiveParameterValue), 'dbs11690318', 3306, '', 0)
#2 E:\en desarrollo\upn212evaldoc\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 E:\en desarrollo\upn212evaldoc\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#4 E:\en desarrollo\upn212evaldoc\system\Database\BaseConnection.php(1408): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#5 E:\en desarrollo\upn212evaldoc\system\Database\BaseConnection.php(1427): CodeIgniter\Database\BaseConnection->listTables()
#6 E:\en desarrollo\upn212evaldoc\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#7 E:\en desarrollo\upn212evaldoc\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#8 E:\en desarrollo\upn212evaldoc\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#9 E:\en desarrollo\upn212evaldoc\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#10 E:\en desarrollo\upn212evaldoc\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#11 E:\en desarrollo\upn212evaldoc\spark(97): CodeIgniter\CLI\Console->run()
#12 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for db5013982481.hosting-data.io failed: No such host is known.  in E:\en desarrollo\upn212evaldoc\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 E:\en desarrollo\upn212evaldoc\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('db5013982481.ho...', 'dbu1124368', Object(SensitiveParameterValue), 'dbs11690318', 3306, '', 0)
#1 E:\en desarrollo\upn212evaldoc\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 E:\en desarrollo\upn212evaldoc\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 E:\en desarrollo\upn212evaldoc\system\Database\BaseConnection.php(1408): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 E:\en desarrollo\upn212evaldoc\system\Database\BaseConnection.php(1427): CodeIgniter\Database\BaseConnection->listTables()
#5 E:\en desarrollo\upn212evaldoc\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 E:\en desarrollo\upn212evaldoc\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 E:\en desarrollo\upn212evaldoc\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 E:\en desarrollo\upn212evaldoc\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 E:\en desarrollo\upn212evaldoc\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 E:\en desarrollo\upn212evaldoc\spark(97): CodeIgniter\CLI\Console->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for db5013982481.hosting-data.io failed: No such host is known.  in E:\en desarrollo\upn212evaldoc\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 E:\en desarrollo\upn212evaldoc\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 E:\en desarrollo\upn212evaldoc\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 E:\en desarrollo\upn212evaldoc\system\Database\BaseConnection.php(1408): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 E:\en desarrollo\upn212evaldoc\system\Database\BaseConnection.php(1427): CodeIgniter\Database\BaseConnection->listTables()
#4 E:\en desarrollo\upn212evaldoc\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 E:\en desarrollo\upn212evaldoc\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 E:\en desarrollo\upn212evaldoc\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#7 E:\en desarrollo\upn212evaldoc\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#8 E:\en desarrollo\upn212evaldoc\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#9 E:\en desarrollo\upn212evaldoc\spark(97): CodeIgniter\CLI\Console->run()
#10 {main}
ERROR - 2024-05-18 13:20:33 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'un212evaldoc' in E:\en desarrollo\upn212evaldoc\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 E:\en desarrollo\upn212evaldoc\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'un212evaldoc', 3306, '', 0)
#1 E:\en desarrollo\upn212evaldoc\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 E:\en desarrollo\upn212evaldoc\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 E:\en desarrollo\upn212evaldoc\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 E:\en desarrollo\upn212evaldoc\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 E:\en desarrollo\upn212evaldoc\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 E:\en desarrollo\upn212evaldoc\app\Validation\Userrules.php(10): CodeIgniter\BaseModel->first()
#7 E:\en desarrollo\upn212evaldoc\system\Validation\Validation.php(312): App\Validation\Userrules->validateUser('12345678', 'email,password', Array, NULL, 'password')
#8 E:\en desarrollo\upn212evaldoc\system\Validation\Validation.php(170): CodeIgniter\Validation\Validation->processRules('password', 'password', '12345678', Array, Array)
#9 E:\en desarrollo\upn212evaldoc\system\Controller.php(140): CodeIgniter\Validation\Validation->run()
#10 E:\en desarrollo\upn212evaldoc\app\Controllers\UserController.php(27): CodeIgniter\Controller->validate(Array, Array)
#11 E:\en desarrollo\upn212evaldoc\system\CodeIgniter.php(934): App\Controllers\UserController->login()
#12 E:\en desarrollo\upn212evaldoc\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#13 E:\en desarrollo\upn212evaldoc\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#14 E:\en desarrollo\upn212evaldoc\public\index.php(67): CodeIgniter\CodeIgniter->run()
#15 E:\en desarrollo\upn212evaldoc\system\Commands\Server\rewrite.php(47): require_once('E:\\en desarroll...')
#16 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'un212evaldoc' in E:\en desarrollo\upn212evaldoc\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 E:\en desarrollo\upn212evaldoc\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 E:\en desarrollo\upn212evaldoc\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 E:\en desarrollo\upn212evaldoc\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 E:\en desarrollo\upn212evaldoc\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 E:\en desarrollo\upn212evaldoc\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 E:\en desarrollo\upn212evaldoc\app\Validation\Userrules.php(10): CodeIgniter\BaseModel->first()
#6 E:\en desarrollo\upn212evaldoc\system\Validation\Validation.php(312): App\Validation\Userrules->validateUser('12345678', 'email,password', Array, NULL, 'password')
#7 E:\en desarrollo\upn212evaldoc\system\Validation\Validation.php(170): CodeIgniter\Validation\Validation->processRules('password', 'password', '12345678', Array, Array)
#8 E:\en desarrollo\upn212evaldoc\system\Controller.php(140): CodeIgniter\Validation\Validation->run()
#9 E:\en desarrollo\upn212evaldoc\app\Controllers\UserController.php(27): CodeIgniter\Controller->validate(Array, Array)
#10 E:\en desarrollo\upn212evaldoc\system\CodeIgniter.php(934): App\Controllers\UserController->login()
#11 E:\en desarrollo\upn212evaldoc\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#12 E:\en desarrollo\upn212evaldoc\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 E:\en desarrollo\upn212evaldoc\public\index.php(67): CodeIgniter\CodeIgniter->run()
#14 E:\en desarrollo\upn212evaldoc\system\Commands\Server\rewrite.php(47): require_once('E:\\en desarroll...')
#15 {main}
CRITICAL - 2024-05-18 13:20:33 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'un212evaldoc'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Validation\Userrules.php(10): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\Validation\Validation.php(312): App\Validation\Userrules->validateUser('12345678', 'email,password', [...], null, 'password')
 7 SYSTEMPATH\Validation\Validation.php(170): CodeIgniter\Validation\Validation->processRules('password', 'password', '12345678', [...], [...])
 8 SYSTEMPATH\Controller.php(140): CodeIgniter\Validation\Validation->run()
 9 APPPATH\Controllers\UserController.php(27): CodeIgniter\Controller->validate([...], [...])
10 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\UserController->login()
11 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
12 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('E:\\en desarrollo\\upn212evaldoc\\public\\index.php')
ERROR - 2024-05-18 13:20:43 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'un212evaldoc' in E:\en desarrollo\upn212evaldoc\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 E:\en desarrollo\upn212evaldoc\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'un212evaldoc', 3306, '', 0)
#1 E:\en desarrollo\upn212evaldoc\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 E:\en desarrollo\upn212evaldoc\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 E:\en desarrollo\upn212evaldoc\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 E:\en desarrollo\upn212evaldoc\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 E:\en desarrollo\upn212evaldoc\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 E:\en desarrollo\upn212evaldoc\app\Validation\Userrules.php(10): CodeIgniter\BaseModel->first()
#7 E:\en desarrollo\upn212evaldoc\system\Validation\Validation.php(312): App\Validation\Userrules->validateUser('deae880618', 'email,password', Array, NULL, 'password')
#8 E:\en desarrollo\upn212evaldoc\system\Validation\Validation.php(170): CodeIgniter\Validation\Validation->processRules('password', 'password', 'deae880618', Array, Array)
#9 E:\en desarrollo\upn212evaldoc\system\Controller.php(140): CodeIgniter\Validation\Validation->run()
#10 E:\en desarrollo\upn212evaldoc\app\Controllers\UserController.php(27): CodeIgniter\Controller->validate(Array, Array)
#11 E:\en desarrollo\upn212evaldoc\system\CodeIgniter.php(934): App\Controllers\UserController->login()
#12 E:\en desarrollo\upn212evaldoc\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#13 E:\en desarrollo\upn212evaldoc\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#14 E:\en desarrollo\upn212evaldoc\public\index.php(67): CodeIgniter\CodeIgniter->run()
#15 E:\en desarrollo\upn212evaldoc\system\Commands\Server\rewrite.php(47): require_once('E:\\en desarroll...')
#16 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'un212evaldoc' in E:\en desarrollo\upn212evaldoc\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 E:\en desarrollo\upn212evaldoc\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 E:\en desarrollo\upn212evaldoc\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 E:\en desarrollo\upn212evaldoc\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 E:\en desarrollo\upn212evaldoc\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 E:\en desarrollo\upn212evaldoc\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 E:\en desarrollo\upn212evaldoc\app\Validation\Userrules.php(10): CodeIgniter\BaseModel->first()
#6 E:\en desarrollo\upn212evaldoc\system\Validation\Validation.php(312): App\Validation\Userrules->validateUser('deae880618', 'email,password', Array, NULL, 'password')
#7 E:\en desarrollo\upn212evaldoc\system\Validation\Validation.php(170): CodeIgniter\Validation\Validation->processRules('password', 'password', 'deae880618', Array, Array)
#8 E:\en desarrollo\upn212evaldoc\system\Controller.php(140): CodeIgniter\Validation\Validation->run()
#9 E:\en desarrollo\upn212evaldoc\app\Controllers\UserController.php(27): CodeIgniter\Controller->validate(Array, Array)
#10 E:\en desarrollo\upn212evaldoc\system\CodeIgniter.php(934): App\Controllers\UserController->login()
#11 E:\en desarrollo\upn212evaldoc\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#12 E:\en desarrollo\upn212evaldoc\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 E:\en desarrollo\upn212evaldoc\public\index.php(67): CodeIgniter\CodeIgniter->run()
#14 E:\en desarrollo\upn212evaldoc\system\Commands\Server\rewrite.php(47): require_once('E:\\en desarroll...')
#15 {main}
CRITICAL - 2024-05-18 13:20:43 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'un212evaldoc'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Validation\Userrules.php(10): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\Validation\Validation.php(312): App\Validation\Userrules->validateUser('deae880618', 'email,password', [...], null, 'password')
 7 SYSTEMPATH\Validation\Validation.php(170): CodeIgniter\Validation\Validation->processRules('password', 'password', 'deae880618', [...], [...])
 8 SYSTEMPATH\Controller.php(140): CodeIgniter\Validation\Validation->run()
 9 APPPATH\Controllers\UserController.php(27): CodeIgniter\Controller->validate([...], [...])
10 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\UserController->login()
11 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
12 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('E:\\en desarrollo\\upn212evaldoc\\public\\index.php')
INFO - 2024-05-18 13:21:12 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-05-18 13:21:12 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'un212evaldoc' in E:\en desarrollo\upn212evaldoc\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 E:\en desarrollo\upn212evaldoc\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'un212evaldoc', 3306, '', 0)
#1 E:\en desarrollo\upn212evaldoc\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 E:\en desarrollo\upn212evaldoc\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 E:\en desarrollo\upn212evaldoc\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 E:\en desarrollo\upn212evaldoc\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 E:\en desarrollo\upn212evaldoc\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 E:\en desarrollo\upn212evaldoc\app\Validation\Userrules.php(10): CodeIgniter\BaseModel->first()
#7 E:\en desarrollo\upn212evaldoc\system\Validation\Validation.php(312): App\Validation\Userrules->validateUser('deae880618', 'email,password', Array, NULL, 'password')
#8 E:\en desarrollo\upn212evaldoc\system\Validation\Validation.php(170): CodeIgniter\Validation\Validation->processRules('password', 'password', 'deae880618', Array, Array)
#9 E:\en desarrollo\upn212evaldoc\system\Controller.php(140): CodeIgniter\Validation\Validation->run()
#10 E:\en desarrollo\upn212evaldoc\app\Controllers\UserController.php(27): CodeIgniter\Controller->validate(Array, Array)
#11 E:\en desarrollo\upn212evaldoc\system\CodeIgniter.php(934): App\Controllers\UserController->login()
#12 E:\en desarrollo\upn212evaldoc\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#13 E:\en desarrollo\upn212evaldoc\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#14 E:\en desarrollo\upn212evaldoc\public\index.php(67): CodeIgniter\CodeIgniter->run()
#15 E:\en desarrollo\upn212evaldoc\system\Commands\Server\rewrite.php(47): require_once('E:\\en desarroll...')
#16 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'un212evaldoc' in E:\en desarrollo\upn212evaldoc\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 E:\en desarrollo\upn212evaldoc\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 E:\en desarrollo\upn212evaldoc\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 E:\en desarrollo\upn212evaldoc\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 E:\en desarrollo\upn212evaldoc\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 E:\en desarrollo\upn212evaldoc\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 E:\en desarrollo\upn212evaldoc\app\Validation\Userrules.php(10): CodeIgniter\BaseModel->first()
#6 E:\en desarrollo\upn212evaldoc\system\Validation\Validation.php(312): App\Validation\Userrules->validateUser('deae880618', 'email,password', Array, NULL, 'password')
#7 E:\en desarrollo\upn212evaldoc\system\Validation\Validation.php(170): CodeIgniter\Validation\Validation->processRules('password', 'password', 'deae880618', Array, Array)
#8 E:\en desarrollo\upn212evaldoc\system\Controller.php(140): CodeIgniter\Validation\Validation->run()
#9 E:\en desarrollo\upn212evaldoc\app\Controllers\UserController.php(27): CodeIgniter\Controller->validate(Array, Array)
#10 E:\en desarrollo\upn212evaldoc\system\CodeIgniter.php(934): App\Controllers\UserController->login()
#11 E:\en desarrollo\upn212evaldoc\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#12 E:\en desarrollo\upn212evaldoc\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 E:\en desarrollo\upn212evaldoc\public\index.php(67): CodeIgniter\CodeIgniter->run()
#14 E:\en desarrollo\upn212evaldoc\system\Commands\Server\rewrite.php(47): require_once('E:\\en desarroll...')
#15 {main}
CRITICAL - 2024-05-18 13:21:12 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'un212evaldoc'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Validation\Userrules.php(10): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\Validation\Validation.php(312): App\Validation\Userrules->validateUser('deae880618', 'email,password', [...], null, 'password')
 7 SYSTEMPATH\Validation\Validation.php(170): CodeIgniter\Validation\Validation->processRules('password', 'password', 'deae880618', [...], [...])
 8 SYSTEMPATH\Controller.php(140): CodeIgniter\Validation\Validation->run()
 9 APPPATH\Controllers\UserController.php(27): CodeIgniter\Controller->validate([...], [...])
10 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\UserController->login()
11 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
12 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('E:\\en desarrollo\\upn212evaldoc\\public\\index.php')
INFO - 2024-05-18 13:21:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-05-18 13:21:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-05-18 13:21:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-05-18 13:21:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-05-18 13:22:03 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
WARNING - 2024-05-18 13:22:03 --> [DEPRECATED] Creation of dynamic property App\Controllers\Admin\UsuarioController::$session is deprecated in APPPATH\Controllers\Admin\UsuarioController.php on line 19.
 1 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\Admin\UsuarioController->__construct()
 2 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('E:\\en desarrollo\\upn212evaldoc\\public\\index.php')
INFO - 2024-05-18 13:22:07 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
WARNING - 2024-05-18 13:22:07 --> [DEPRECATED] Creation of dynamic property App\Controllers\Admin\UsuarioController::$session is deprecated in APPPATH\Controllers\Admin\UsuarioController.php on line 19.
 1 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\Admin\UsuarioController->__construct()
 2 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('E:\\en desarrollo\\upn212evaldoc\\public\\index.php')
INFO - 2024-05-18 13:22:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
WARNING - 2024-05-18 13:22:44 --> [DEPRECATED] Creation of dynamic property App\Controllers\Admin\UsuarioController::$session is deprecated in APPPATH\Controllers\Admin\UsuarioController.php on line 19.
 1 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\Admin\UsuarioController->__construct()
 2 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('E:\\en desarrollo\\upn212evaldoc\\public\\index.php')
INFO - 2024-05-18 13:22:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
WARNING - 2024-05-18 13:22:44 --> [DEPRECATED] Creation of dynamic property App\Controllers\Admin\UsuarioController::$session is deprecated in APPPATH\Controllers\Admin\UsuarioController.php on line 19.
 1 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\Admin\UsuarioController->__construct()
 2 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('E:\\en desarrollo\\upn212evaldoc\\public\\index.php')
INFO - 2024-05-18 13:22:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-05-18 13:23:08 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-05-18 13:23:08 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-05-18 13:23:12 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-05-18 13:23:12 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-05-18 13:25:56 --> Class "App\Controllers\UserModel" not found
in APPPATH\Controllers\ResetPassword.php on line 19.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\ResetPassword->sendResetLink()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ResetPassword))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('E:\\en desarrollo\\upn212evaldoc\\public\\index.php')
ERROR - 2024-05-18 13:26:22 --> Email: sendWithMail throwed ErrorException: mail(): Failed to connect to mailserver at &quot;localhost&quot; port 25, verify your &quot;SMTP&quot; and &quot;smtp_port&quot; setting in php.ini or use ini_set() in E:\en desarrollo\upn212evaldoc\system\Email\Email.php:1735
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mail(): Failed ...', 'E:\\en desarroll...', 1735)
#1 E:\en desarrollo\upn212evaldoc\system\Email\Email.php(1735): mail('edgardegantea@o...', '=?UTF-8?Q?Recup...', 'Haga clic en el...', 'Date: Sat, 18 M...', '-f edgar.degant...')
#2 E:\en desarrollo\upn212evaldoc\system\Email\Email.php(1667): CodeIgniter\Email\Email->sendWithMail()
#3 E:\en desarrollo\upn212evaldoc\system\Email\Email.php(1565): CodeIgniter\Email\Email->spoolEmail()
#4 E:\en desarrollo\upn212evaldoc\app\Controllers\ResetPassword.php(39): CodeIgniter\Email\Email->send()
#5 E:\en desarrollo\upn212evaldoc\system\CodeIgniter.php(934): App\Controllers\ResetPassword->sendResetLink()
#6 E:\en desarrollo\upn212evaldoc\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ResetPassword))
#7 E:\en desarrollo\upn212evaldoc\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 E:\en desarrollo\upn212evaldoc\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 E:\en desarrollo\upn212evaldoc\system\Commands\Server\rewrite.php(47): require_once('E:\\en desarroll...')
#10 {main}
ERROR - 2024-05-18 13:26:22 --> Email: Unable to send email using PHP mail(). Your server might not be configured to send mail using this method.
ERROR - 2024-05-18 13:26:22 --> 
INFO - 2024-05-18 13:26:22 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-05-18 13:26:22 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-05-18 13:27:25 --> Email: sendWithMail throwed ErrorException: mail(): Failed to connect to mailserver at &quot;localhost&quot; port 25, verify your &quot;SMTP&quot; and &quot;smtp_port&quot; setting in php.ini or use ini_set() in E:\en desarrollo\upn212evaldoc\system\Email\Email.php:1735
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mail(): Failed ...', 'E:\\en desarroll...', 1735)
#1 E:\en desarrollo\upn212evaldoc\system\Email\Email.php(1735): mail('edgar.degante.a...', '=?UTF-8?Q?Recup...', 'Haga clic en el...', 'Date: Sat, 18 M...', '-f edgar.degant...')
#2 E:\en desarrollo\upn212evaldoc\system\Email\Email.php(1667): CodeIgniter\Email\Email->sendWithMail()
#3 E:\en desarrollo\upn212evaldoc\system\Email\Email.php(1565): CodeIgniter\Email\Email->spoolEmail()
#4 E:\en desarrollo\upn212evaldoc\app\Controllers\ResetPassword.php(39): CodeIgniter\Email\Email->send()
#5 E:\en desarrollo\upn212evaldoc\system\CodeIgniter.php(934): App\Controllers\ResetPassword->sendResetLink()
#6 E:\en desarrollo\upn212evaldoc\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ResetPassword))
#7 E:\en desarrollo\upn212evaldoc\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 E:\en desarrollo\upn212evaldoc\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 E:\en desarrollo\upn212evaldoc\system\Commands\Server\rewrite.php(47): require_once('E:\\en desarroll...')
#10 {main}
ERROR - 2024-05-18 13:27:25 --> Email: Unable to send email using PHP mail(). Your server might not be configured to send mail using this method.
ERROR - 2024-05-18 13:27:25 --> 
INFO - 2024-05-18 13:27:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-05-18 13:27:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-05-18 17:29:08 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-05-18 17:30:11 --> Email: sendWithMail throwed ErrorException: mail(): Failed to connect to mailserver at &quot;localhost&quot; port 25, verify your &quot;SMTP&quot; and &quot;smtp_port&quot; setting in php.ini or use ini_set() in E:\en desarrollo\upn212evaldoc\system\Email\Email.php:1735
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mail(): Failed ...', 'E:\\en desarroll...', 1735)
#1 E:\en desarrollo\upn212evaldoc\system\Email\Email.php(1735): mail('edgardegantea@o...', '=?UTF-8?Q?Recup...', 'Haga clic en el...', 'Date: Sat, 18 M...', '-f edgar.degant...')
#2 E:\en desarrollo\upn212evaldoc\system\Email\Email.php(1667): CodeIgniter\Email\Email->sendWithMail()
#3 E:\en desarrollo\upn212evaldoc\system\Email\Email.php(1565): CodeIgniter\Email\Email->spoolEmail()
#4 E:\en desarrollo\upn212evaldoc\app\Controllers\ResetPassword.php(39): CodeIgniter\Email\Email->send()
#5 E:\en desarrollo\upn212evaldoc\system\CodeIgniter.php(934): App\Controllers\ResetPassword->sendResetLink()
#6 E:\en desarrollo\upn212evaldoc\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ResetPassword))
#7 E:\en desarrollo\upn212evaldoc\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 E:\en desarrollo\upn212evaldoc\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 E:\en desarrollo\upn212evaldoc\system\Commands\Server\rewrite.php(47): require_once('E:\\en desarroll...')
#10 {main}
ERROR - 2024-05-18 17:30:11 --> Email: Unable to send email using PHP mail(). Your server might not be configured to send mail using this method.
ERROR - 2024-05-18 17:30:11 --> 
INFO - 2024-05-18 17:30:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-05-18 17:30:12 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-05-18 17:35:09 --> Email: sendWithMail throwed ErrorException: mail(): Failed to connect to mailserver at &quot;localhost&quot; port 25, verify your &quot;SMTP&quot; and &quot;smtp_port&quot; setting in php.ini or use ini_set() in E:\en desarrollo\upn212evaldoc\system\Email\Email.php:1735
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mail(): Failed ...', 'E:\\en desarroll...', 1735)
#1 E:\en desarrollo\upn212evaldoc\system\Email\Email.php(1735): mail('edgar.degante.a...', '=?UTF-8?Q?Recup...', 'Haga clic en el...', 'Date: Sat, 18 M...', '-f edgar.degant...')
#2 E:\en desarrollo\upn212evaldoc\system\Email\Email.php(1667): CodeIgniter\Email\Email->sendWithMail()
#3 E:\en desarrollo\upn212evaldoc\system\Email\Email.php(1565): CodeIgniter\Email\Email->spoolEmail()
#4 E:\en desarrollo\upn212evaldoc\app\Controllers\ResetPassword.php(39): CodeIgniter\Email\Email->send()
#5 E:\en desarrollo\upn212evaldoc\system\CodeIgniter.php(934): App\Controllers\ResetPassword->sendResetLink()
#6 E:\en desarrollo\upn212evaldoc\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ResetPassword))
#7 E:\en desarrollo\upn212evaldoc\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 E:\en desarrollo\upn212evaldoc\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 E:\en desarrollo\upn212evaldoc\system\Commands\Server\rewrite.php(47): require_once('E:\\en desarroll...')
#10 {main}
ERROR - 2024-05-18 17:35:09 --> Email: Unable to send email using PHP mail(). Your server might not be configured to send mail using this method.
ERROR - 2024-05-18 17:35:09 --> 
INFO - 2024-05-18 17:35:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-05-18 17:35:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-05-18 17:41:40 --> Email: sendWithMail throwed ErrorException: mail(): Failed to connect to mailserver at &quot;localhost&quot; port 25, verify your &quot;SMTP&quot; and &quot;smtp_port&quot; setting in php.ini or use ini_set() in E:\en desarrollo\upn212evaldoc\system\Email\Email.php:1735
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mail(): Failed ...', 'E:\\en desarroll...', 1735)
#1 E:\en desarrollo\upn212evaldoc\system\Email\Email.php(1735): mail('edgar.degante.a...', '=?UTF-8?Q?Recup...', 'Haga clic en el...', 'Date: Sat, 18 M...', '-f edgar.degant...')
#2 E:\en desarrollo\upn212evaldoc\system\Email\Email.php(1667): CodeIgniter\Email\Email->sendWithMail()
#3 E:\en desarrollo\upn212evaldoc\system\Email\Email.php(1565): CodeIgniter\Email\Email->spoolEmail()
#4 E:\en desarrollo\upn212evaldoc\app\Controllers\ResetPassword.php(39): CodeIgniter\Email\Email->send()
#5 E:\en desarrollo\upn212evaldoc\system\CodeIgniter.php(934): App\Controllers\ResetPassword->sendResetLink()
#6 E:\en desarrollo\upn212evaldoc\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ResetPassword))
#7 E:\en desarrollo\upn212evaldoc\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 E:\en desarrollo\upn212evaldoc\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 E:\en desarrollo\upn212evaldoc\system\Commands\Server\rewrite.php(47): require_once('E:\\en desarroll...')
#10 {main}
ERROR - 2024-05-18 17:41:40 --> Email: Unable to send email using PHP mail(). Your server might not be configured to send mail using this method.
ERROR - 2024-05-18 17:41:40 --> 
INFO - 2024-05-18 17:41:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-05-18 17:41:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-05-18 17:44:42 --> Email: sendWithMail throwed ErrorException: mail(): Failed to connect to mailserver at &quot;localhost&quot; port 25, verify your &quot;SMTP&quot; and &quot;smtp_port&quot; setting in php.ini or use ini_set() in E:\en desarrollo\upn212evaldoc\system\Email\Email.php:1735
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mail(): Failed ...', 'E:\\en desarroll...', 1735)
#1 E:\en desarrollo\upn212evaldoc\system\Email\Email.php(1735): mail('edgardegantea@o...', '=?UTF-8?Q?Recup...', 'Haga clic en el...', 'Date: Sat, 18 M...', '-f edgar.degant...')
#2 E:\en desarrollo\upn212evaldoc\system\Email\Email.php(1667): CodeIgniter\Email\Email->sendWithMail()
#3 E:\en desarrollo\upn212evaldoc\system\Email\Email.php(1565): CodeIgniter\Email\Email->spoolEmail()
#4 E:\en desarrollo\upn212evaldoc\app\Controllers\ResetPassword.php(39): CodeIgniter\Email\Email->send()
#5 E:\en desarrollo\upn212evaldoc\system\CodeIgniter.php(934): App\Controllers\ResetPassword->sendResetLink()
#6 E:\en desarrollo\upn212evaldoc\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ResetPassword))
#7 E:\en desarrollo\upn212evaldoc\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 E:\en desarrollo\upn212evaldoc\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 E:\en desarrollo\upn212evaldoc\system\Commands\Server\rewrite.php(47): require_once('E:\\en desarroll...')
#10 {main}
ERROR - 2024-05-18 17:44:42 --> Email: Unable to send email using PHP mail(). Your server might not be configured to send mail using this method.
ERROR - 2024-05-18 17:44:42 --> 
INFO - 2024-05-18 17:44:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-05-18 17:44:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
